content/docs/manage/organizations.md

```diff
@@ -37,27 +37,72 @@ Here's a breakdown of what actions each role can take:
+  <div style={{ flex: '0 0 55%', marginTop: '-20px' }}>
+    <img src="/docs/manage/orgs_create_button.png" alt="create organization button" style={{ verticalAlign: 'middle', maxWidth: '100%' }} />
+  </div>
+</div>
 
-![create organization button](/docs/manage/orgs_create_button.png)
+### Free Plan
 
-You'll be asked to select a plan and confirm your billing details. After confirming billing, you'll be directed to your organization's billing page, where you can take steps like inviting [members](/docs/manage/orgs-manage#invite-members) and [transferring projects](/docs/manage/orgs-project-transfer).
+If you're on the Free Plan, you'll need to choose a paid plan for your organization (since organizations are a paid feature).
+
+<div style={{ display: 'flex', alignItems: 'top' }}>
+  <div style={{ flex: '0 0 45%', paddingRight: '20px' }}>
+
+    You'll be prompted to choose to select a plan and enter billing details. After confirming, you'll be directed to your organization's billing page, where you can invite [members](/docs/manage/orgs-manage#invite-members) and [transfer projects](/docs/manage/orgs-project-transfer).
+
+  </div>
+  <div style={{ flex: '0 0 55%', marginTop: '-20px' }}>
 
 ![getting started with new org](/docs/manage/orgs_create_next.png)
 
+  </div>
+</div>
+
+### Paid plans
+
+<div style={{ display: 'flex', alignItems: 'top' }}>
+  <div style={{ flex: '0 0 45%', paddingRight: '20px' }}>
+    If you're already on a paid plan for your personal account, you'll have two options: 
+    - convert your personal account to an organization
+    - create a new organization from scratch.
+  </div>
+  <div style={{ flex: '0 0 55%', marginTop: '-20px' }}>
+
+![convert personal account to an organization](/docs/manage/orgs_create_with_transfer.png)
+
+  </div>
+</div>
+
+#### Convert your personal account
+
+Converting your personal account transfers all of your projects and billing to the new organization. Your personal account will switch to the Free Plan. This is a quick, one-time, one-way operation: everything in your personal account is transferred to your new organization instantly. There is no service disruption, no change to your connections, and no way to move resources back to your personal account after the conversion.
+
+#### Create a new organization
+
+If you choose to create a new organization, you can decide which projects (if any) you want to [transfer](/docs/manage/orgs-project-transfer). First, you'll need to select a paid plan for the organization. Your personal account will remain on its current paid plan — so note that you'll be billed for both your personal and organization accounts. This option is helpful if you want to keep personal projects separate while still requiring paid features like higher allowances or additional security.
+
+<Admonition type="note">
+Project transfers to an organization are one-way. You cannot move organization projects back to your personal account.
+</Admonition>
+
 ## What's next?
```

Comment:
Does this section redundant now that there's self-serve orgs? If you follow the self-serve instructioons, would these next steps still apply?

Reply:
Yes, good point. I've moved relevant content into the main create flow and removed the section.